packages/core/scripts/voting-upgrade-umip/3_Verify.js

```diff
@@ -37,7 +38,7 @@ async function runExport() {
 
   const newVotingContract = await Voting.at(votingAddress);
   const currentOwner = await newVotingContract.owner();
-  assert.equal(web3.utils.toChecksumAddress(currentOwner), web3.utils.toChecksumAddress(Governor.address));
+  assert.equal(web3.utils.toChecksumAddress(currentOwner), web3.utils.toChecksumAddress(governor.address));
```

Comment:
curious question: why was this change required? if you are running this with truffle can you not pull the address directly from the truffle artefact?

Reply:
I think this broke when we moved over to using our own `getTruffleArtifact` method. `.address` is seeded by some async method being called on the instance so it's able to grab the network id from web3. Truffle does this for it's artifacts on startup, but since this hadn't been done yet for the artifact that we created, the instance doesn't know what address to grab.